app/uk/gov/hmrc/merchandiseinbaggage/views/DeclarationConfirmationView.scala.html

```diff
@@ -66,10 +66,8 @@ <h2 id="whatToDoNextId" class="govuk-heading-m">@messages("declarationConfirmati
   }
   <li>@messages(s"declarationConfirmation.ul.2.${journeyType.entryName}")</li>
 
-  @for(calcResult <- declaration.maybeTotalCalculationResult) {
-    @if(calcResult.calculationResults.proofOfOriginNeeded && declaration.declarationType == Import) {
-      <li>@messages("declarationConfirmation.ul.3")</li>
-    }
+  @if( ViewUtils.proofOfOriginNeeded(declaration) && declaration.declarationType == Import) {
```

Comment:
this may be an existing bit already, but the `proofOfOriginNeeded` method should consider the `declarationType`for completeness instead of keeping that check extra here in the `if`

Reply:
done